src/bg/help/guides/getting-started.md

```diff
@@ -0,0 +1,105 @@
+   <el-tag type="warning">Не приложението от Windows Store!</el-tag>
+1. Изтеглете и инсталирайте [iCloud за Windows 8](https://support.apple.com/en-us/HT204283).
+   <el-tag type="warning">Не приложението от Windows Store!</el-tag>
+	> Не намирате линка за изтегляне на **iCloud** за Windows 8? [GIF](https://imgur.com/a/P1ef4Wd)
+1. Рестартирайте компютъра си, след като сте инсталирали **iTunes** и **iCloud**.
+   <el-tag type="warning">Трябва да рестартирате компютъра си!</el-tag>
+1. Екстрактнете изтегления **`AltInstaller.zip`** файл. Отворете **`setup.exe`** и следвайте инструкциите.
+1. Включете **AltServer** от *Известия*.
+	> Не намирате **AltServer**? [Image](https://imgur.com/a/rSagfh2)
+1. Свържете Вашия **iPhone** или **iPad** с компютъра и се уверете, че е отключен и устройството Ви е доверено.
+1. Отворете **iTunes** и включете "**iTunes Wi-Fi синхронизация**" за дадения iPhone/iPad.
+1. Натиснете **AltServer** иконката в *Известия* и намерете "**Istall AltStore**, след което изберете Вашия iPhone/iPad.
+1. Въведете Вашите Apple ID имейл и парола. (**внимавайте** за малки и главни букви).
+	> *Вашето Apple ID и парола се изпращат само на Apple и никъде другаде.*
+1. AltStore би трябвало да бъде инсталиран на Вашия iPhone/iPad.
+
+### Paperback
+#### Инсталация
+1. Отворете **AltServer** заедно с **iCloud** и **iTunes**.
+1. Уверете се, че сте на _една и съща мрежа_ (един и същи рутер в повечете случаи) с компютъра, на който работи **AltServer**. Ако не сте на една мрежа, може да свържете устройството си чрез кабел.
+1. Доверете AltStore: На устройството си отворете Settings > General > Device Management. Изберете своето Apple ID и изберете **Довери**.
+1. Отворете **AltStore** на Вашия iPhone/iPad.
+1. Влезте в **My Apps** и натиснете **+** иконката в горния ляв ъгъл.
+1. Отворете **`.IPA`** файла, който изтеглихте от секцията [Предварителни изисквания](/help/guides/getting-started/#prerequisites).
+1. **Paperback** би трябвало да бъде инсталиран!
+:::
+::::
+
+## Инсталация ЧЗВ
+### Получавам "Access Denied", когато се опитвам да инсталирам AltStore
+Отворете AltServer като администратор и се уверете, че сте рестартирали компютъра си, след като сте инсталирали iTunes и iCloud.
+
+### Не мога да намеря .IPA файла, който трябва да изтегля
+Може да изтеглите **`.IPA`** файла <Download text="от тук"/>
+
+### Имам друг проблем, който не е описан тук
+Вижте в [AltStore FAQ](https://altstore.io/faq/). Ако още не може да разрешите проблема, посетете #support канала на нашия **[Discord](https://discord.gg/Ny83JV3)** за бърз отговор или направете пост в **[Reddit](https://www.reddit.com/r/Paperback/)**.
+
+---
+
+## Помощ за приложението
+Ако изпитвате проблеми с приложението, моля, проверете **[Помощ](/help/faq/#troubleshooting)**. Ако изпитвате проблем, който не е споменат в тази страница, моля, използвайте канала #support в **[Discord](https://discord.gg/Ny83JV3)**, за да зададете въпроса си.
```

Comment:
You should link the translated page instead of the English one.
```suggestion
Ако изпитвате проблеми с приложението, моля, проверете **[Помощ](bg/help/faq/#troubleshooting)**. Ако изпитвате проблем, който не е споменат в тази страница, моля, използвайте канала #support в **[Discord](https://discord.gg/Ny83JV3)**, за да зададете въпроса си.
```